src/main/java/org/openbmp/db_rest/resources/LinkState.java

```diff
@@ -145,132 +145,162 @@ else if (peerHashId != null)
                 DbUtils.select_DbToJson(mysql_ds, query));
     }
 
+    @GET
+    @Path("/nodes/virtual_nodes/peer/{peerHashId}/{mt_id}")
```

Comment:
The PR does not indicate this change is making link-state changes.  Break these into different pull requests.